airbyte-integrations/connectors/destination-gcs/src/test-integration/java/io/airbyte/integrations/destination/gcs/GcsAvroTestDataComparator.java

```diff
@@ -33,4 +36,23 @@ protected boolean compareDateTimeValues(String airbyteMessageValue, String desti
     return super.compareDateTimeValues(airbyteMessageValue, format.format(dateTime));
   }
 
+  @Override
+  protected boolean compareTime(final String airbyteMessageValue, final String destinationValue) {
+    var destinationDate = LocalTime.ofInstant(getInstantFromEpoch(destinationValue), ZoneOffset.UTC);
```

Comment:
Minor nit: instead of `var` have this be `LocalTime` so it's clear that the difference between this `super.compareTime` and this child method is that this is doing a `LocalTime` comparison vs `String` comparison